aiida/cmdline/params/options/__init__.py

```diff
@@ -324,6 +324,8 @@ def decorator(command):
     help='Only include entries with this process state.'
 )
 
+PAUSED = OverridableOption('--paused', 'paused', is_flag=True, help='Only include entries that are paused.')
```

Comment:
Running out of letters lol; I see `p` and `P` are already taken

Reply:
I figured this might not be used _that_ often so it's okay to only have the long form, which is not that long anyway